tests/pregenerated_data_util_test.go

```diff
@@ -378,7 +378,7 @@ func NewBCDataForPreGeneratedTest(testDataDir string, tc *preGeneratedTC) (*BCDa
 	engine := istanbulBackend.New(genesisAddr, istanbul.DefaultConfig, validatorPrivKeys[0], chainDB, gov, node.CONSENSUSNODE)
 
 	////////////////////////////////////////////////////////////////////////////////
-	// Make a BlockChain
+	// Make a blockChain
```

Comment:
Why is this changed?

Reply:
Looks like Goland's refactoring feature has changed it automatically...